Dockerfile

```diff
@@ -1,28 +1,18 @@
-FROM ubuntu:14.04
-MAINTAINER Vaibhav Bhembre <vaibhav@digitalocean.com>
-
-ENV GOROOT /goroot
-ENV GOPATH /go
-ENV PATH $GOROOT/bin:$PATH
-ENV APPLOC $GOPATH/src/github.com/digitalocean/ceph_exporter
-
-RUN apt-get install -y apt-transport-https
-
-RUN echo "deb https://download.ceph.com/debian-jewel trusty main" >> /etc/apt/sources.list
+FROM ubuntu:16.04
 
 RUN apt-get update && \
-    apt-get install -y build-essential git curl
+    apt-get install -y --no-install-recommends apt-transport-https ca-certificates && \
+    echo "deb https://download.ceph.com/debian-jewel xenial main" >> /etc/apt/sources.list && \
+    apt-key adv --keyserver ha.pool.sks-keyservers.net --recv E84AC2C0460F3994 && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends git golang gcc libc6-dev librados-dev librbd-dev
```

Comment:
It helps to decouple the Go version from the one that Ubuntu's package manager defaults to so that we can explicitly track it. I think we should be able to up the version to `v1.6` as a separate PR.